src/ZendeskApi.Client/Resources/TagsResource.cs

```diff
@@ -0,0 +1,28 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using ZendeskApi.Client.Models;
+using ZendeskApi.Client.Resources.Interfaces;
+using ZendeskApi.Client.Responses;
+
+namespace ZendeskApi.Client.Resources
+{
+    public class TagsResource : AbstractBaseResource<TagsResource>, ITagsResource
+    {
+        private static string ResourceUri = "/api/v2/tags";
+        public TagsResource(IZendeskApiClient apiClient, ILogger logger) : base(apiClient, logger, "tags")
+        {
+        }
+
+        public async Task<TagResponse> GetAllAsync(CursorPager cursor = null, CancellationToken cancellationToken = default)
```

Comment:
I think the other cursor endpoint return `ICursorPagination<T>`. We should probably do the same here for consistency. See here for another example: https://github.com/justeat/ZendeskApiClient/blob/5d8af4deb12ddae55973985d33112ef68283b48b/src/ZendeskApi.Client/Resources/Ticket/TicketsResource.cs#L46C17-L46C17

Reply:
Thanks! all makes sense! good for another look 🙏